docs/output.md

```diff
@@ -591,7 +593,34 @@ The script included in the pipeline uses DESeq2 to normalise read counts across
 
-The PCA plots are generated based alternately on the top five hundred most variable genes, or all genes. The former is the conventional approach that is more likely to pick up strong effects (ie the biological signal) and the latter, when different, is picking up a weaker but consistent effect that is synchronised across many transcripts.  We project both of these onto the first two PCs (shown in the top row of the figure below), which is where most of the information about the samples is captured, but we elso evaluate other components' in light of their predicitity of a samples _condition_: we plot (bottom left, below) the proportion of variability due to condition that is explained by each component, and do an additional PC plot on the two components that best explain _condition_ (bottom right, below - where both the first two PCs happened to be the best explanations of variability in _condition_). If _condition_ was arbitrary, or a compromise to capture a multivariable design, then this mightn't be optimal, but in simple cases might reveal a more complex picture than simply the first two PCs.
+The PCA plots are generated based alternately on the top five hundred
+most variable genes, or all genes. The former is the conventional
+approach that is more likely to pick up strong effects (ie the
+biological signal) and the latter, when different, is picking up a
+weaker but consistent effect that is synchronised across many
+transcripts.  We project both of these onto the first two PCs (shown
+in the top row of the figure below), which is the best two dimensional
+representation of the variation between samples.
+
+We also explore higher components in terms of experimental factors
+inferred from sample names. If your sample naming convention follows a
+strict policy of using underscores to delimit values of experimental
+factors (for example `WT_UNTREATED_REP1`) and all names have the same
+number of underscores (so excluding `WT_TREATED_10ml_REP1` from being
+compatible with the previous label), then any of these factors that
+are informative (ie label some but not all samples the same) then we
+individually plot upto the first five PCs, per experimental level, for
+each of the experimental factors.
+
+The plot on the left hand side shows the standard PC plot - notice the
+variable number of underscores, meaning that the central plot would
+not be produced: here we have changed the underscore that is
+hyphenating the treatment to a '-' character. This allows the central
+plot to be generated, and we can see that replicate (the 2nd part of
+the sample name) seems to be affecting the 3rd principal component,
+but the treatment factor is affecting the more important first two
+components. The right-most plot shows all pairwise euclidean distances
+between the samples.
```

Comment:
No limit on the number of character in a simple line, but just these paragraphs have one, it's a bit disturbing...

Reply:
Sorted for consistency 👍🏽 https://github.com/nf-core/rnaseq/pull/617/commits/1ef84f8848846d8f8b5759de29dfe93f2c077359